robot_skills/README.md

```diff
@@ -1,6 +1,17 @@
 # robot_skills
 
-Robot_skills provide interfaces to all parts of a robot: its base, arms, head, perception, worldmodel, speech system, etc. 
-The package also provides a console for a robot, so you can give the robot commands like goto(<some coordinate>) via a console
+`robot_skills` provide interfaces to all parts of a robot: its base, arms, head, perception, worldmodel, speech system, etc.
 
 A robot is represented via a robot-object which can be used by the [robot_smach_states](https://github.com/tue-robotics/tue_robocup/tree/master/robot_smach_states)
+
+This package contains a [`mockbot`](src/robot_skills/mockbot.py), which expose all the needed interfaces.
+
+## Robot registration
+
+To implement your own robot, create a robot specific skills package. To register your robot add the following to the `package.xml` of your package:
+```xml
+<export>
+  <robot_skills robot="bla_skills.bla.Bla" />
+</export>
+```
```

Comment:
It would also be valuable to refer to the 'unittest'. I'm planning on expanding this so that in principle if your robot passes the test, it can be used in smach states.